repose-aggregator/components/filters/cors-filter/src/main/scala/org/openrepose/filters/cors/CorsFilter.scala

```diff
@@ -44,18 +48,18 @@ class CorsFilter @Inject()(configurationService: ConfigurationService)
 
   import CorsFilter._
 
-  private var configurationFile: String = DEFAULT_CONFIG
+  private var configurationFile: String = _
   private var initialized = false
-  private var allowedOrigins: Iterable[Regex] = _
-  private var allowedMethods: Iterable[String] = _
-  private var resources: Iterable[Resource] = _
+  private var allowedOrigins: Seq[Regex] = _
+  private var allowedMethods: Seq[String] = _
+  private var resources: Seq[Resource] = _
```

Comment:
It mostly stemmed from wanting to change this:
`validMethods.exists(requestedMethod == _)`
to this:
`validMethods.contains(method)`
where `requestedMethod` was renamed to `method`.

From there it allowed me to propagate the change throughout.  I vaguely remember having another reason, but that was the big one.